frontend/src/store/queries/forgot-password.ts

```diff
@@ -0,0 +1,32 @@
+import { ENV } from '@common/enums/app/env.enum';
+import { ApiPath, AuthApiPath, ContentType } from '@common/enums/enums';
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+type ResetPasswordRequestData = {
+  id: string;
+  password: string;
+};
+
+export const forgotPasswordApi = createApi({
+  reducerPath: 'forgotPasswordApi',
+  baseQuery: fetchBaseQuery({ baseUrl: ENV.API_PATH }),
+  endpoints: (builder) => ({
+    forgotPassword: builder.mutation<void, string>({
+      query: (email) => ({
+        url: `${ApiPath.AUTH}/local${AuthApiPath.RESET_PASSWORD_REQUEST}/${email}`,
+        method: 'POST',
+      }),
+    }),
+    resetPassword: builder.mutation<void, ResetPasswordRequestData>({
+      query: (payload) => ({
+        url: `${ApiPath.AUTH}/local${AuthApiPath.RESET_PASSWORD}`,
+        method: 'POST',
+        contentType: ContentType.JSON,
+        body: payload,
+      }),
+    }),
+  }),
+});
```

Comment:
Please follow the one-root-createApi rule @o-rudych shared here - https://binary-academy-2022.slack.com/archives/C03SA9P93C6/p1660658147966669